src/constants/locales.js

```diff
@@ -0,0 +1,186 @@
+    {"code":"oc","name":"Occitan","nativeName":"Occitan"},
+    {"code":"oj","name":"Ojibwe, Ojibwa","nativeName":"ᐊᓂᔑᓈᐯᒧᐎᓐ"},
+    {"code":"cu","name":"Old Church Slavonic, Church Slavic, Church Slavonic, Old Bulgarian, Old Slavonic","nativeName":"ѩзыкъ словѣньскъ"},
+    {"code":"om","name":"Oromo","nativeName":"Afaan Oromoo"},
+    {"code":"or","name":"Oriya","nativeName":"ଓଡ଼ିଆ"},
+    {"code":"os","name":"Ossetian, Ossetic","nativeName":"ирон æвзаг"},
+    {"code":"pa","name":"Panjabi, Punjabi","nativeName":"ਪੰਜਾਬੀ, پنجابی‎"},
+    {"code":"pi","name":"Pāli","nativeName":"पाऴि"},
+    {"code":"fa","name":"Persian","nativeName":"فارسی"},
+    {"code":"pl","name":"Polish","nativeName":"polski"},
+    {"code":"ps","name":"Pashto, Pushto","nativeName":"پښتو"},
+    {"code":"pt","name":"Portuguese","nativeName":"Português"},
+    {"code":"qu","name":"Quechua","nativeName":"Runa Simi, Kichwa"},
+    {"code":"rm","name":"Romansh","nativeName":"rumantsch grischun"},
+    {"code":"rn","name":"Kirundi","nativeName":"kiRundi"},
+    {"code":"ro","name":"Romanian, Moldavian, Moldovan","nativeName":"română"},
+    {"code":"ru","name":"Russian","nativeName":"русский язык"},
+    {"code":"sa","name":"Sanskrit (Saṁskṛta)","nativeName":"संस्कृतम्"},
+    {"code":"sc","name":"Sardinian","nativeName":"sardu"},
+    {"code":"sd","name":"Sindhi","nativeName":"सिन्धी, سنڌي، سندھی‎"},
+    {"code":"se","name":"Northern Sami","nativeName":"Davvisámegiella"},
+    {"code":"sm","name":"Samoan","nativeName":"gagana faa Samoa"},
+    {"code":"sg","name":"Sango","nativeName":"yângâ tî sängö"},
+    {"code":"sr","name":"Serbian","nativeName":"српски језик"},
+    {"code":"gd","name":"Scottish Gaelic; Gaelic","nativeName":"Gàidhlig"},
+    {"code":"sn","name":"Shona","nativeName":"chiShona"},
+    {"code":"si","name":"Sinhala, Sinhalese","nativeName":"සිංහල"},
+    {"code":"sk","name":"Slovak","nativeName":"slovenčina"},
+    {"code":"sl","name":"Slovene","nativeName":"slovenščina"},
+    {"code":"so","name":"Somali","nativeName":"Soomaaliga, af Soomaali"},
+    {"code":"st","name":"Southern Sotho","nativeName":"Sesotho"},
+    {"code":"es","name":"Spanish; Castilian","nativeName":"español, castellano"},
+    {"code":"su","name":"Sundanese","nativeName":"Basa Sunda"},
+    {"code":"sw","name":"Swahili","nativeName":"Kiswahili"},
+    {"code":"ss","name":"Swati","nativeName":"SiSwati"},
+    {"code":"sv","name":"Swedish","nativeName":"svenska"},
+    {"code":"ta","name":"Tamil","nativeName":"தமிழ்"},
+    {"code":"te","name":"Telugu","nativeName":"తెలుగు"},
+    {"code":"tg","name":"Tajik","nativeName":"тоҷикӣ, toğikī, تاجیکی‎"},
+    {"code":"th","name":"Thai","nativeName":"ไทย"},
+    {"code":"ti","name":"Tigrinya","nativeName":"ትግርኛ"},
+    {"code":"bo","name":"Tibetan Standard, Tibetan, Central","nativeName":"བོད་ཡིག"},
+    {"code":"tk","name":"Turkmen","nativeName":"Türkmen, Түркмен"},
+    {"code":"tl","name":"Tagalog","nativeName":"Wikang Tagalog, ᜏᜒᜃᜅ᜔ ᜆᜄᜎᜓᜄ᜔"},
```

Comment:
I think we can use Dhivehi and Tagalog as native names to avoid the cryptic squares